webapp/packages/core-resource/src/Resource/ResourceAlias.ts

```diff
@@ -42,6 +42,10 @@ export abstract class ResourceAlias<TKey, TOptions extends ResourceAliasOptions>
     return undefined;
   }
 
+  copy(): this {
```

Comment:
I think we should add some comments here to explain why we need this method

Reply:
removed